src/NetAnalyzers/Core/Microsoft.NetCore.Analyzers/MicrosoftNetCoreAnalyzersResources.resx

```diff
@@ -1510,4 +1510,14 @@
     <value>and all other platforms</value>
     <comment>This call site is reachable on: 'windows' 10.0.2000 and later, and all other platforms</comment>
   </data>
+  <data name="ProvideStreamMemoryBasedAsyncOverridesDescription" xml:space="preserve">
+    <value>The default implementation of '{2}' delegates to '{1}', which is innefficient as it requires an extra copy. It also results in extra allocations as it returns Task instead of ValueTask. Consider moving {0}'s implementation to '{2}' and have '{1}' delegate to '{2}'.</value>
```

Comment:
```suggestion
    <value>The default implementation of '{2}' delegates to '{1}', which is innefficient as it requires an extra copy. It also results in extra allocations as it returns 'Task' instead of 'ValueTask'. Consider moving {0}'s implementation to '{2}' and have '{1}' delegate to '{2}'.</value>
```

Reply:
Actually, you cannot have format arguments `{0}`, `{1}`, etc. in description string, it is a static description per-descriptor, not something instantiated per-diagnostic. You should instead follow what @carlossanlop did in a similar analyzer and use more generic description string that is applicable for all violations.

Reply:
Fixed.